Cyra/lab2/flake.nix

```diff
@@ -0,0 +1,19 @@
+{
+  description = "Test Flake";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs?ref=nixos-unstable";
+  };
+
+  outputs = { self, nixpkgs }: {
+    packages = {
+      aarch64-darwin = let
+        pkgs = nixpkgs.legacyPackages.aarch64-darwin;
+      in
+      {
+        hello = pkgs.hello;
+        default = pkgs.callPackage ./default.nix {};
+      };
+    };
+  };
+}
```

Comment:
this flake is missing a proper use of nix packages, see:

https://github.com/nixvital/nix-based-cpp-starterkit/blob/main/flake.nix